tests/utils.py

```diff
@@ -6,13 +6,22 @@ def get_test_filepath(filepath):
     return os.path.join(curr_file_dirpath, filepath)
 
 
+def get_filename_without_ext(path):
+    filename = os.path.basename(path)
+    return os.path.splitext(filename)[0]
+
+
 def prepare_feature_and_py_files(testdir, feature_file, py_file):
```

Comment:
This function was used only once, and it's implementation was tied to the caller. I would say to remove it and let the caller `test_scenario_in_expanded_mode` test manage the resources (and let the test be in the pytest-bdd style, not a normal test)

Reply:
I will rewrite it to pytest-bdd style.

I added `prepare_feature_and_py_files` to be able to reuse feature files that are already in other tests, to test them in expanded mode.

I will remove it, and I'll create separate tests for gherkin reporter. That means that we may have some duplication but it is safer approach.

Reply:
Tests has been rewritten to pytest-bdd style.